### DISCUSSION TITLE
Query

### DISCUSSION THREAD

**Discussion by coderuhaan2004** [RapidAI/RapidLatexOCR#24] [Ideas]
Wanted to know, whether this project just converts the snapshots of math expressions into LaTeX or it takes the image of whole page including text and math and extract math and converts it to LaTeX?

I really wanted the latter feature.

**Comment by SWHL** [comment]
The project is the former, which only convert the snapshots of math expressions to LaTeX.

If you want to the latter, you need to find a formula detector. You can refer this [link](https://www.breezedeus.com/article/p2t-mfd-v1.1.1)
